### ISSUE TITLE
Couldn't connect to Discord via RPC: Error: Could not connect [DELETED]

### ISSUE THREAD

**Issue by NikoNiyazi** [iCrawl/discord-vscode#1541]
why did it my issue get deleted out of blue lol
---------------------------------------------------
tried stable and canary discord
i need help

**Comment by Jacksonmills** [comment]
Yeah same issue, no ability to connect to discord anymore. I am using WSL: Ubuntu vscode

**Comment by snoweuph** [comment]
exactly same error for me on Linux mint using the flatpak version of vscode

**Comment by snoweuph** [comment]
Okay the Fix is actually Quite simple: neither use vscode or discord through flatpak, because flatpak is sand boxing everything (on its own a good thing, but for this actually bad) after removing the flatpaks and instead installing the Debian packages it works perfect again

**Comment by NikoNiyazi** [comment]
> Okay the Fix is actually Quite simple: neither use vscode or discord through flatpak, because flatpak is sand boxing everything (on its own a good thing, but for this actually bad) after removing the flatpaks and instead installing the Debian packages it works perfect again

i use windows

**Comment by snoweuph** [comment]
okay, can't help for windows, just can say that for me on linux the problem laid in the sandboxing of flatpak
